src/jukebox/components/rpc_command_alias.py

```diff
@@ -57,6 +63,24 @@
         'plugin': 'ctrl',
         'method': 'toggle',
         'ignore_card_removal_action': True},
+    'toggle_shuffle': {
```

Comment:
Do we may also need the option to explicitly set a mode?
This way the user can bind the de-/activation to different cards, which could sometimes be easier to control.

The command could be `shuffle` with modes `toggle`, `activate`, `deactivate` ?